website/project/model.py

```diff
@@ -298,6 +328,13 @@ def create(cls, auth, **kwargs):
             save=False,
         )
 
+        comment.new_mentions = get_valid_mentioned_users_guids(comment, comment.node.contributors)
```

Comment:
Rather than storing `new_mentions` on `comment`, can we pass it as an additional argument to the `{comment,mention}_added` signals?